tests/fake_has_transformed_position/test_fake_has_transformed_position.py

```diff
@@ -20,33 +20,66 @@ def test_transform():
     c = yaqc.Client(38001)
     c.set_native_reference(1.0)
     val = 3.0
-    assert np.isclose(c.to_native(val), 2.5)
     bb = c.to_transformed(c.to_native(val))
     assert np.isclose(val, bb)
 
 
 @testing.run_daemon_entry_point("fake-has-transformed-position", config=config)
 def test_limits():
+    for port in [38001, 38002]:
+        c = yaqc.Client(port)
+        config = tomli.loads(c.get_config())
+
+        limits = c.get_limits()
+
+        for setpoint, actual in zip(
+            [limits[0] - 0.5, 0.5 * sum(limits), limits[1] + 0.5],
+            [limits[0], 0.5 * sum(limits), limits[1]],
+        ):
+            c.set_position(setpoint)
+            time.sleep(0.10)
+            assert np.isclose(c.get_position(), actual)
+            assert np.isclose(c.get_native_position(), c.to_native(actual))
+
+        lims2 = list(map(c.to_transformed, c.get_native_limits()))
+        assert np.isclose(limits, lims2).all()
+
```

Comment:
See bottom comment on why it might be useful to test state values such as destination.

Reply:
tests included destination checks now